content/03.methods.md

```diff
@@ -1,9 +1,33 @@
-#Materials and Methods
+# Materials and Methods
+## Hetionet
+Hetionet [@doi:10.7554/eLife.26726] is a large heterogenous network that contains pharmacological and biological information.
+This network depicts information in the form of nodes and edges of different types: nodes that represent biological and pharmacological entities and edges which represent relationships each entity may share with itself or another. 
```

Comment:
"relationships between entities" may be more clear than "relationships each entity may share with itself or another". The "share with itself" phrasing makes me think of self-loops. There are no self loops in Hetionet v1.0. Try the following Cypher query at https://neo4j.het.io/browser/:

```cypher
MATCH path=(source)-[rel]->(target)
WHERE source = target
RETURN type(rel), count(rel)
```